varinfo/umm_var.py

```diff
@@ -226,9 +226,6 @@ def get_dimension_information(
     * Projected horizontal spatial dimension (no UMM-Var type for this).
     * Swath dimensions (along or across track).
 
-    Also note: VariableFromDmr instances do not currently extract variable
-    shapes.
-
```

Comment:
Nice - I appreciate you keeping this documentation up-to-date, particularly as it's not really in the main area of the code change.